nodes/blobstream-bootstrapper.md

```diff
@@ -0,0 +1,74 @@
+# Blobstream bootstrapper
+
+To bootstrap the Blobstream P2P network, we use the bootstrapper Blobstream
+node type to accept connections from freshly created orchestrators/relayers
+and share its peer table with them.
+
+## How to run
+
+### Install the Blobstream binary
+
+Make sure to have the Blobstream binary installed. Check
+[the Blobstream binary page](https://docs.celestia.org/nodes/blobstream-binary)
+for more details.
+
+### Init the store
+
+Before starting the bootstrapper, we will need to init the store:
+
+```sh
+blobstream bootstrapper init
+```
+
+By default, the store will be created un `~/.bootstrapper`. However,
+if you want to specify a custom location, you can use the `--home` flag.
+Or, you can use the following environment variable:
+
+<!-- markdownlint-disable MD013 -->
+
+| Variable            | Explanation                         | Default value     | Required |
+| ------------------- | ----------------------------------- | ----------------- | -------- |
+| `BOOTSTRAPPER_HOME` | Home directory for the bootstrapper | `~/.bootstrapper` | Optional |
+
+### Add keys
+
+The P2P private key is optional, and a new one will be generated automatically
+on the start if none is provided.
+
+The `p2p` sub-command will help you set up this key if you want to use a specific
+one:
+
+```sh
+blobstream bootstrapper p2p  --help
+```
+
+### Start the bootstrapper
+
+Now that we have the store initialized, we can start the bootstrapper:
+
+```shell
+blobstream bootstrapper
+
+Blobstream P2P network bootstrapper command
+
+Usage:
+  blobstream bootstrapper [command]
+
+Aliases:
+  bootstrapper, bs
+
+Flags:
+  -h, --help   help for bootstrapper
+
+Use "blobstream bootstrapper [command] --help" for more information about a command.
+```
```

Comment:
[blocking] should this help output describe the available commands? Why isn't `start` explicitly mentioned?

When I run this command on my machine, I see:

```
$ blobstream bootstrapper
Blobstream P2P network bootstrapper command

Usage:
  blobstream bootstrapper [command]

Aliases:
  bootstrapper, bs

Available Commands:
  init        Initialize the Blobstream bootstrapper store. Passed flags have persisted effect.
  p2p         Blobstream p2p keys manager
  start       Starts the bootstrapper node using the provided home.Could be connected to other bootstrapper nodes too.

Flags:
  -h, --help   help for bootstrapper

Use "blobstream bootstrapper [command] --help" for more information about a command.
```

instead of copying the help output here, this section of the docs could state

```suggestion
blobstream bootstrapper start
```

Reply:
https://github.com/jcstein/orchestrator-relayer/commit/883d1c83a1ace91fb34c24034fd4167d6bb657ba

Reply:
will update after 606 is merged on orch-relayer